scripts/generate_instrumentation_bootstrap.py

```diff
@@ -91,7 +92,16 @@ def main():
     with open(gen_path, "w") as gen_file:
         gen_file.write(source)
 
-    subprocess.run(["black", "-q", gen_path], check=True)
+    subprocess.run(
```

Comment:
Curious as to why we are running `black` on this folder?

Reply:
Generator just writes out valid, functioning python code without having to worry about formatting. We then use black+isort to make sure generated code is properly formatted.  Forcing the generator to care about formatting world unnecessarily complicated it.